src/frontend/komponenter/Fagsak/journalposter/JournalpostListe.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from 'react';
+
+import styled from 'styled-components';
+
+import { Element } from 'nav-frontend-typografi';
+
+import { useHttp } from '@navikt/familie-http';
+import { RessursStatus } from '@navikt/familie-typer';
+
+import { IPersonInfo } from '../../../typer/person';
+import { DagMånedÅr, tilVisning } from '../../../utils/kalender';
+
+import 'nav-frontend-tabell-style';
+
+const Container = styled.div`
+    margin: 1rem;
+`;
+
+interface IProps {
+    bruker: IPersonInfo;
+}
+
+interface Journalpost {
+    journalpostId: string;
+    tittel: string;
+    behandlingstema: string;
+    datoMottatt: DagMånedÅr;
+    tema: string;
+    journalstatus: string;
+}
+
+const JournalpostListe: React.FC<IProps> = ({ bruker }) => {
+    const { request } = useHttp();
+    const [journalposter, settjournalposter] = useState<Journalpost[]>([]);
+
+    useEffect(() => {
+        request<undefined, Journalpost[]>({
```

Comment:
Kall `settJournalposter(byggHenterRessurs())` før vi gjør kallet for å kunne vise spinner mens vi henter data.